assets/js/strict.js

```diff
@@ -1 +1,2 @@
+// @flow
```

Comment:
line break after (or just don't even add `@flow` to this file, lol)

Reply:
I know what you mean. I just wanted to make `flow coverage` happy.